packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -1169,7 +1169,7 @@ export default class CashierStore extends BaseStore {
             'fees',
             this.config.account_transfer.selected_to.currency,
         ]);
-        this.config.account_transfer.transfer_fee = typeof transfer_fee === 'undefined' ? 1 : +transfer_fee;
+        this.config.account_transfer.transfer_fee = typeof transfer_fee === 'undefined' ? 0 : +transfer_fee;
```

Comment:
I think You can right this line simpler:

```suggestion
        this.config.account_transfer.transfer_fee = Number(transfer_fee || 0);
```

Reply:
This is a great solution @mahdiaryayi-fs :) , however I noticed that it will not provide protection from undefined.
As far as I know, typeof is safer as it allows the identifier to never have been declared before. Please see a screenshot below:
<img width="781" alt="Screenshot 2021-09-09 at 11 57 19" src="https://user-images.githubusercontent.com/87321516/132655941-88a1be24-e1d3-4434-8e2f-df73839e0eba.png">

How should I proceed in your opinion?

Reply:
@maryia-deriv it will happen if `transfer_fee` is not defined at all. but in your case, that's defined on line 1166 so it won't happen in this case